sdk/identity/identity/src/msal/utils.ts

```diff
@@ -221,15 +221,16 @@ export function handleMsalError(
   return new AuthenticationRequiredError({ scopes, getTokenOptions, message: error.message });
 }
 
-// transformations.ts
-
-export function publicToMsal(account: AuthenticationRecord): msalCommon.AccountInfo {
-  const [environment] = account.authority.match(/([a-z]*\.[a-z]*\.[a-z]*)/) || [""];
-  return {
-    ...account,
-    localAccountId: account.homeAccountId,
-    environment,
-  };
+// transformations
+export function publicToMsal(account: AuthenticationRecord): msalCommon.AccountInfo | null {
```

Comment:
👋 do we have any tests for this function? Could we add some? Would be nice to get some coverage and ensure we don't regress

Reply:
Thanks Maor. We have now changed our approach a little bit, and not doing any transformations b/w authority and environment